### ISSUE TITLE
Log off with backpack in curio slot causes glitched backpack

### ISSUE THREAD

**Issue by Glemoyn** [Tiviacz1337/Travelers-Backpack#238]
**Edit: figured out the bug (I believe). to recreate, you place the backpack on your back with the button in the GUI, which does not place it in the curio slot, and log off. Then you change the config to require the player use the curio slot, and when you log on, you can no longer open the backpack with the b button, which means you can not remove it from your back. and since it is not in the curio slot, you cant remove it. I dont know if changing the server settings back will allow the bag to be removed with the GUI button.**

So I logged off of my server with the backpack in my back curio slot, when I logged in the backpack was missing from my inventory altogether, but still showed on my back. Drowning did not drop the backpack on the ground, and when I create a new backpack and place it in the curio slot, it shows the tank contents but nothing else (I had 2 picks in the tool slots). doing the /kill command dropped the new bag and removed the glitched bag from the bag (did not give the glitched bag back) but the new bag no longer appeared on the bag when in curio slot, and cant open the bag when in the curio slot (the new backpack was created via creative if that matters)

When opening the new backpack when on the hot bar it shows an empty inventory. putting items in the inventory of the new back pack show when on the back, but the tank contents will show the glitched packs tank. This is not true for the tool slots, the ones you place in the new bag show in the GUI, but not on the character.

screen shot of the backpack that glitched out
![2021-01-26_14 21 56](https://user-images.githubusercontent.com/49622844/105915459-9dd38780-5fe4-11eb-87dd-a35cfef84c34.png)

screen shot of the curio slot empty with the backpack tank still visible on character
![2021-01-26_14 22 09](https://user-images.githubusercontent.com/49622844/105915476-a3c96880-5fe4-11eb-9309-46f4075f0850.png)

Picture of the new backpacks contents
![2021-01-26_14 23 04](https://user-images.githubusercontent.com/49622844/105915490-a75cef80-5fe4-11eb-87fc-e35103304c07.png)

picture of the backpack contents while in curio slot
![2021-01-26_14 23 45](https://user-images.githubusercontent.com/49622844/105915499-aa57e000-5fe4-11eb-8e81-a973b03683b7.png)

**Required informations (please complete the following information):**
 - Mod version: 1.16.4-5.3.8
 - curios forge (version 1.16.4-4.0.3.5)
 - Minecraft v 1.16.5
 - other mods on Server : Ice and fire, Ambient Sounds, astral, quark, Biomes o plenty, bloodmagic, botania, create, , Dark utilities, dynamic surroundings, ftb chunks, ftb gui lib, immersive engineering, jei, jei intergration, jeresources, morpheous, optifine 1.16.5, pams hc food core, storage drawers, theoneprobe, corail tombstone, waystones

**Comment by Glemoyn** [comment]
As an update, the config file was set that you had to use the curio slot, when that was set to false, the button that allowed the player to access the backpack inventory on your back worked again, and the backpack no longer was able to be in the curio slot.
I dont know if that is how it is meant to be, the way I read the config option was if it was set to false, you could use the curio slot if you want but wasnt required.

**Comment by Tiviacz1337** [comment]
Well, there's no automatical swap between Curios integration. If you enable it, normal way of wearing backpack will be broken, if you disable it, then you can't put it in the Curios slot

**Comment by Tiviacz1337** [comment]
Also, I could theoretically make glitched backpack _unequipable_, but what if someone use it as second storage instead. If you have any good approaches for it, I would like to know about them. As for now, I won't do anything with that.

**Comment by Glemoyn** [comment]
so I never got the backpack that was glitched off my back. I think the glitch was just it still on the players back when settings were changed to curio only, so if we had reverted the settings the player would have been able to get the backpack off with the gui buttons. But the player died to try and get the backpack to drop on the ground, and that just deleted the backpack it seems.

I would say add a warning above the curios only config saying something like this "If enabled, any backpacks on players backs using the buttons in the GUI will be unable to be removed and unable to open. Make sure all players have the backpacks in their inventory, not on their backs." or something of the sort.
